### PR TITLE
fix spelling errors in bash

### PR DESCRIPTION
**Pull request by liamzebedee**
#1104

### CODE REVIEW

File: bash.html.markdown [hunk 1]
```diff
@@ -56,7 +56,7 @@ echo '$Variable'
 
 # String substitution in variables
 echo ${Variable/Some/A}
-# This will substitute the first occurance of "Some" with "A"
+# This will substitute the first occurence of "Some" with "A"
```
Comment (geoffliu) [inline bash.html.markdown]: occurrence :)

Comment (liamzebedee) [inline bash.html.markdown]: `s/`:smile:`/`:stuck_out_tongue:`/g`

Comment (geoffliu) [inline bash.html.markdown]: ?

**Comment by levibostian** [comment]
@liamzebedee mind fixing the spelling mistake above so we can merge this in? Change "occurence" to "occurrence"

**Comment by liamzebedee** [comment]
@levibostian there you go

**Comment by geoffliu** [comment]
Thanks!

**Pull request merged**